opal/util/show_help.c

```diff
@@ -88,6 +104,50 @@ static void opal_show_help_finalize(void)
     }
 }
 
+static void opal_show_help_cbfunc(pmix_status_t status, void *cbdata)
+{
+    opal_log_info_t *info = (opal_log_info_t *) cbdata;
+    if(PMIX_SUCCESS != status && PMIX_OPERATION_SUCCEEDED != status) {
+        opal_output(output_stream, "%s", info->msg);
```

Comment:
Are you ignoring the opal_help_want_aggregate value in case of error here? This is probably worth a comment explaining why.

Reply:
Yes, I  figured it would be fine to print it out rather than not. At this point the aggregate feature won't work. I'll add a comment.